howso/utilities/feature_attributes/base.py

```diff
@@ -535,6 +535,7 @@ def _process(self,  # noqa: C901
                  dependent_features: Optional[Dict[str, List[str]]] = None,
                  include_sample: bool = False,
                  max_workers: Optional[int] = None,
+                 type_overrides: Optional[dict] = None,
```

Comment:
This should be more strongly typed

Reply:
Additionally, instead of using `str` for the feature type might be better to define a Literal type var to define the valid values.